packages/lodestar-cli/src/cmds/beacon/handler.ts

```diff
@@ -20,7 +20,7 @@ import {initBeaconState} from "./initBeaconState";
  * Run a beacon node
  */
 export async function beaconHandler(args: IBeaconArgs & IGlobalArgs): Promise<void> {
-  await initBLS();
+  await init("blst-native");
```

Comment:
Here use fallback to WASM but if it fails add a fat message of performance warning